frontend/src/pages/study/index.tsx

```diff
@@ -308,53 +308,17 @@ export default function Draft() {
 					</div>
 				</div>
 			);
-	else
-		results = (
-			<div className="mt-1 mr-2 ml-2 p-4 border border-[#c8c8c8] rounded-[16px] transition duration-150 flex">
-				<div>
-					<div
-						className= "text-[0.8rem] text-[#aaaaaa] pb-1 cursor-pointer hover:text-black"
-					>
-						{ genCtxText.length > 100 ? '...' : '' }
-						{ genCtxText.substring(genCtxText.length - 100) }
```

Comment:
this makes `getCtxText` unused, so as the linter suggests, we should remove that.